test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -47,6 +47,9 @@ public void execute() throws IllegalValueException {
      * the result matches the persons in the expectedPersonList exactly.
      */
     private void assertFindCommandBehavior(String[] keywords, List<ReadOnlyPerson> expectedPersonList) {
+        List<String> keywordsList = Arrays.asList(keywords);
+        keywordsList.replaceAll(String::toLowerCase);
+        keywords = keywordsList.toArray(new String[0]);
         FindCommand command = createFindCommand(keywords);
         CommandResult result = command.execute();
```

Comment:
Hmmm in this case since your implementation changes the parser I think it is okay. 

However, because you updated the parser, it is also possible to update the parser tests to reflect that :)